pyirf/irf/energy_dispersion.py

```diff
@@ -155,4 +206,13 @@ def energy_dispersion_to_migration(
 
         migration_matrix[idx, :, :] = y
 
+    with np.errstate(invalid="ignore"):
+        migration_matrix = (
+            migration_matrix
+            / migration_matrix.sum(axis=2).sum(axis=1)[:, np.newaxis, np.newaxis]
```

Comment:
Why is this needed? This looks wrong. Summing over the offset axis is definitely not right here.

Reply:
but you want the probability to be independent of the offset the migration matrix is really a 2D object